cache/in-memory/src/lib.rs

```diff
@@ -705,6 +706,7 @@ impl InMemoryCache {
             joined_at: member.joined_at.to_owned(),
             mute: member.mute,
             nick: member.nick.to_owned(),
+            pending: false,
```

Comment:
I'm not sure about this part. This function is only being referenced by MessageCreate, and if we're getting this event from a user, it means they are not really pending.